src/lib/useKeyfileGenerator.ts

```diff
@@ -242,14 +253,56 @@ export const useSingleKeyfileGenerator = ({
       revision: point.keyRevisionNumber,
     });
 
-    if (Just.hasInstance(derivedSeed)) {
-      setSeeds([derivedSeed.value]);
+    if (Just.hasInstance(defaultDerivedSeed)) {
+      setDefaultSeeds([defaultDerivedSeed.value]);
+    }
+
+    /**
+     * Otherwise, we will need to try a different token derivation scheme.
+     *
+     * Due to a logical error with auth token derivation, keys set between May
+     * 2021 and March 2022 used `sha256` instead of the logically correct
+     * `keccak256`. To mitigate this for users who log back into Bridge to
+     * download their keyfile or copy their access code, we need to check for
+     * the existence by deriving with a fallback algorithm.
```

Comment:
The "otherwise" here doesn't flow well with the previous comment. I'd do "if we can't" or something similar.

Also, this fails to mention that this issue only affected custodial wallets. Probably worth calling out explicitly, even (or especially?) if the code below does so too.

Reply:
This phrasing was not clear and potentially confusing, updated it to reflect the actual expected behavior.